pages/workHeader.xml

```diff
@@ -12,450 +12,458 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+               to works in standardized lists, such as Chaîne's Répertoire or various publications
+               listing the single Miracles of Mary can be given as a bibliographic reference in
+                  <gi>listBibl</gi>, while references to external sources such as the <ref
+                  target="https://corpuscoranicum.de/kontexte/uebersicht">Umwelttexte by Corpus
+                  Coranicum</ref> can be given in a <gi>relation</gi> between the record and the
+               external reference in <gi>listRelation</gi>, see below.</p>
+            <p>If there is a need to assign an external clavis ID to textparts of a work, this
+               should be done within <tag>div type="edition"</tag>. A Clavis ID should be assigned
+               for each textpart inside <gi>listBible</gi>:</p>
+            <egXML xmlns="http://www.tei-c.org/ns/Examples">
+               <div type="textpart" xml:id="Theophilos2">
+                  <label>Extract from the 21st Festal letter (Graf 1937, 383)</label>
+                  <listBibl type="clavis">
+                     <bibl type="CPG">
+                        <ptr target="bm:CPG2"/>
+                        <citedRange unit="item">2589</citedRange>
+                     </bibl>
+                  </listBibl>
+               </div>
+            </egXML>
+         </div>
+
+         <div type="level2">
+            <head>Witnesses</head>
+
+            <p>As stated in <ref target="https://github.com/BetaMasaheft/Documentation/issues/714"
+                  >issue 714</ref>, we always use <gi>witness</gi> for those manuscripts which are
+               used for the edition we are going to (or could potentially) provide in the <tag>div
+                  type='edition'</tag>, if one click 'text' in the upper menu. All manuscript
+               records containing the work if they have been fully encoded with the correct
+               reference are displayed in the app anyway. However, since not all manuscript records
+               have been properly encoded yet, following <ref
+                  target="https://github.com/BetaMasaheft/Documentation/issues/2690">issue
+                  2690</ref> and <ref
+                  target="https://github.com/BetaMasaheft/Documentation/issues/2699">issue
+                  2699</ref>, if someone wants to list further records containing this work, an
+               acceptable practice is to list otherwise invisible manuscripts as <gi>witness</gi> as well.
+           Since identification of witnesses is part of the research, whoever has done it should be acknowledge in the <gi>note</gi> and bibliographic references (if available).</p>
+            <p>If someone decides to list manuscripts that were neither used in the edition nor fall in the category of stubs, for the sake of clarity, using the <gi>note</gi>, one can state what is its specific relevance.</p>
+            <egXML xmlns="http://www.tei-c.org/ns/Examples">
```

Comment:
Is there an example of this use of `<note>` that could be included in the following example?

Reply:
Then maybe better just leave it, I think